zzz_generated/chip-tool/zap-generated/test/Commands.h

```diff
@@ -615,96 +615,79 @@ class Test_TC_BI_2_2 : public TestCommand
     std::atomic_uint16_t mTestIndex;
     const uint16_t mTestCount = 8;
 
-    chip::Callback::Callback<void (*)(void * context, uint8_t status)> mOnFailureCallback_0{ OnFailureCallback_0, this };
-    chip::Callback::Callback<void (*)(void * context, bool presentValue)> mOnSuccessCallback_0{ OnSuccessCallback_0, this };
```

Comment:
Not sure if this is expected, but why other mOnFailureCallback_* is removed by codegen?

Reply:
I see, it was caused by previous unclean merge.

Reply:
Because this was in parallel with #11660 and CI would have failed without that.  I'll rebase on top of that now that it's merged and then this part will be gone.